lib/content/utils.js

```diff
@@ -1,74 +1,126 @@
 'use strict';
 
 const config = require('config');
-const _ = require('lodash');
 const isUUID = require('validator/lib/isUUID');
 
+const workflows = require('../workflows/utils');
 
-const check = {
+/**
+ * Grab type from all content types in system
```

Comment:
For all of these, using promises as a shared interface feels like the right approach so a user doesn't need to have their own checking in place to determine if what's been returned is a valid value or not.